### PR TITLE
Updates /code/game/objects/item/food/* to follow the style guide

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## About The Pull Request

Updates every file in /code/game/objects/item/food/ to follow the style guide.
if i missed something please lmk

**regex used for finding errors**

`,[^ \\]`  # For finding non spaced commas
`[^ +/*-=]=[^ =]` # finding non spaced equal signs
`/[A-Za-z]([, )]|$)` finding single letter variable assignments

## Why It's Good For The **CODEBASE**

i like giving the maintainers a shitload of work (and burning myself out in the process)

## Changelog

Nothing to see here

### CODE REVIEW

File: code/game/objects/items/food/misc.dm [hunk 1]
```diff
@@ -1,6 +1,16 @@
 
 ////////////////////////////////////////////OTHER////////////////////////////////////////////
 
+/obj/item/food/cheese
```
Comment (SuperNovaa41) [inline code/game/objects/items/food/misc.dm:4]: item/food/chese was defined after it's parents and it made me very uncomfortable

File: code/game/objects/items/food/burgers.dm [hunk 2]
```diff
@@ -39,12 +39,13 @@
 /obj/item/food/burger/human/CheckParts(list/parts_list)
 	..()
 	var/obj/item/food/patty/human/human_patty = locate(/obj/item/food/patty/human) in contents
-	if(LAZYLEN(human_patty.custom_materials))
-		for(var/datum/material/meat/mob_meat/mob_meat_material in human_patty.custom_materials)
-			if(mob_meat_material.subjectname)
-				name = "[mob_meat_material.subjectname] burger"
-			else if(mob_meat_material.subjectjob)
-				name = "[mob_meat_material.subjectjob] burger"
+	if(!LAZYLEN(human_patty.custom_materials))
```
Comment (Kylerace) [inline code/game/objects/items/food/burgers.dm:42]: for(var/element in some_list) doesnt iterate if the list is empty or null so theres no need for this length check

File: code/game/objects/items/food/meat.dm [hunk 3]
```diff
@@ -396,32 +396,32 @@
...
 	return MANUAL_SUICIDE
 
-/obj/item/food/monkeycube/proc/finish_suicide(mob/living/M) ///internal proc called by a monkeycube's suicide_act using a timer and callback. takes as argument the mob/living who activated the suicide
-	if(QDELETED(M) || QDELETED(src))
+/obj/item/food/monkeycube/proc/finish_suicide(mob/living/user) ///internal proc called by a monkeycube's suicide_act using a timer and callback. takes as argument the mob/living who activated the suicide
+	if(QDELETED(user) || QDELETED(src))
 		return
-	if((src.loc != M)) //how the hell did you manage this
-		to_chat(M, span_warning("Something happened to [src]..."))
+	if((src.loc != user)) //how the hell did you manage this
```
Comment (Kylerace) [inline code/game/objects/items/food/meat.dm:419]:
```suggestion
	if(src.loc != user) //how the hell did you manage this
```

File: code/game/objects/items/food/meat.dm [hunk 4]
```diff
@@ -1093,7 +1096,7 @@
 /obj/item/food/meat/rawcutlet/plain/human/MakeGrillable()
 	AddComponent(/datum/component/grillable, /obj/item/food/meat/cutlet/plain/human, rand(35 SECONDS, 50 SECONDS), TRUE, TRUE)
 
-/obj/item/food/meat/rawcutlet/plain/human/OnCreatedFromProcessing(mob/living/user, obj/item/I, list/chosen_option, atom/original_atom)
+/obj/item/food/meat/rawcutlet/plain/human/OnCreatedFromProcessing(mob/living/user, obj/item/item, list/chosen_option, atom/original_atom)
 	. = ..()
 	if(istype(original_atom, /obj/item/food/meat))
```
Comment (Kylerace) [inline code/game/objects/items/food/meat.dm:1101]: this proc could use an early return

File: code/game/objects/items/food/misc.dm [hunk 5]
```diff
@@ -400,13 +399,13 @@
...
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/nutriment/vitamin = 1, /datum/reagent/iron = 10, /datum/reagent/consumable/sugar = 5, /datum/reagent/medicine/omnizine = 2) //Honk
-	var/mutable_appearance/head
-	var/headcolor = rgb(0, 0, 0)
 	tastes = list("candy" = 1)
 	foodtypes = JUNKFOOD | SUGAR
 	food_flags = FOOD_FINGER_FOOD
 	slot_flags = ITEM_SLOT_MASK
 	w_class = WEIGHT_CLASS_TINY
+	var/mutable_appearance/head
+	var/headcolor = rgb(0, 0, 0)
```
Comment (Kylerace) [inline code/game/objects/items/food/misc.dm:408]: underscore should separate head and color

**Review by Kylerace — CHANGES_REQUESTED** [review]
4 for the price of 1

**Comment by SuperNovaa41** [comment]
what the freak why didn't it remove the stale tag

**Review by tralezab — APPROVED** [review]
noice... thanks for handlin' kylers review

**Pull request merged**